package.json

```diff
@@ -34,6 +34,7 @@
     "husky": "^6.0.0",
     "lint-staged": "^10.5.4",
     "prettier": "2.2.1",
+    "storybook-css-modules-preset": "^1.1.1",
```

Comment:
You installed this module, but you do not load it:

> add the following plugin on your ./storybook/main.js
` 'storybook-css-modules-preset',`

See https://www.npmjs.com/package/storybook-css-modules-preset

In addition, this change doesn't seem related to "Add Euro Button".